wiki/Beatmap/Pattern/osu!/Bursts/en.md

```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - bursts
+---
+
+# Bursts
+
+**Bursts** are a higly used pattern in [beatmaps](/wiki/Beatmap). They consist of 3 or more [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) that are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on eachother.
+Bursts mostly consist of 1/4 or faster rythms.
```

Comment:
"Bursts are a highly used pattern in beatmaps" Even so, starting it off like this instead of describing what a burst seems redundant, not saying I disagree.

```suggestion
Bursts are commonly snapped to 1/4 ticks.
```